mynewportxps/newportxps/newportxps.py

```diff
@@ -539,6 +539,8 @@ def set_velocity(self, stage, velo, accl=None,
             return
         ret, v_cur, a_cur, jt0_cur, jt1_cur = \
              self._xps.PositionerSGammaParametersGet(self._sid, stage)
+        if velo is None:
```

Comment:
try not to change newportxps

Reply:
is this a hard rule? The only way around this would be to rewrite the entire function inside alicptfts, which I don't think is very useful. Is anyone else using the newportxps.py code? If not, I don't think it hurts to make a small change that also overall improves newportxps